admin/base/settings/defaults.py

```diff
@@ -28,6 +29,14 @@
 # Email settings. Account created for testing. Password shouldn't be hardcoded
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
Just a note for devops, this should be set to `'django.core.mail.backends.smtp.EmailBackend'` in the > dev local.py.